### ISSUE TITLE
Where's Wulfie? Backend with WikiBase 1.3.0 doesn't love Amadeus

### ISSUE THREAD

**Issue by sfelshin** [infolab-csail/WikipediaBase#244]
The symbols and synonyms loaded into Omnibase as part of the new backend on navassa, built to accompany WikipediaBase 1.3.0, contain plenty of Mozarts, but where is Wolfgang Amadeus? Some other classes know him, but there is no `wikibase-term`.  Is this a problem with loading symbols and synonyms into Omnibase?  With calculating symbols and synonyms?  Wikipedia GIGO?
```
$ telnet navassa 8008
(get-known "mozart")
((wikibase-term "mozart" :span (0 6) :name "Mozart, West Virginia" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart (train)" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart the music processor" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart, Saskatchewan" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart!" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "MOZART (model)" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart (horse)" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "MozART group" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart (crater)" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart (com&#233;die musicale)" :priority 0)
 (wikibase-term "mozart" :span (0 6) :name "Mozart (1955 film)" :priority 0)
 (imdb-person "mozart" :span (0 6) :name "Wolfgang Amadeus Mozart" :priority 0)
 (imdb-person "mozart" :span (0 6) :name "Leopold Mozart" :priority 0)
 (imdb-person "mozart" :span (0 6) :name "George Mozart" :priority 0)
 (imdb-movie "mozart" :span (0 6) :name "The Life and Loves of Mozart (1955)" :priority 0)
 (imdb-movie "mozart" :span (0 6) :name "Motsart (2006)" :priority 0)
 (imdb-movie "mozart" :span (0 6) :name "Mozart (2013)" :priority 0)
 (imdb-movie "mozart" :span (0 6) :name "Mozart (2002)" :priority 0)
 (imdb-movie "mozart" :span (0 6) :name "Mozart (1936)" :priority 0)
 (calle-city "mozart" :span (0 6) :name "Mozart, Arkansas, USA" :priority 0)
 (calle-city "mozart" :span (0 6) :name "Mozart, Arkansas, USA (Mozart2)" :priority 0)
 (calle-city "mozart" :span (0 6) :name "Mozart, Idaho, USA" :priority 0)
 (calle-city "mozart" :span (0 6) :name "Mozart, Pennsylvania, USA" :priority 0)
 (calle-city "mozart" :span (0 6) :name "Mozart, West Virginia, USA" :priority 0)
 (gutenberg-author "mozart" :span (0 6) :name "Mozart, Wolfgang Amadeus" :priority 0)
 (classicalnet-composer "mozart" :span (0 6) :name "MOZART, Wolfgang Amadeus" :priority 0)
 (wb-person "mozart" :span (0 6) :name "Mozart, Wolfgang Amadeus" :priority 0))
```

**Comment by michaelsilver** [comment]
@sfelshin The symbols and synonyms generated most recently by WikipediaBase are in `/archive/infolab/backup/WikipediaBase/2016-05-01/wikibase-term-synonyms`. I'm not sure how omnibase uses this information in the case of your query. Otherwise, all other files loaded into omnibase were unchanged in [`github:omnibase/dbase/wikibase-term`](https://github.com/infolab-csail/omnibase/tree/master/dbase/wikibase-term).

**Comment by sfelshin** [comment]
I grepped through `archive/infolab/backup/WikipediaBase/2016-05-01/wikibase-term-synonyms` for `<tab>Mozart$` and didn't find it as a synonym of Wolfgang Amadeus Mozart.  I cross-checked by grepping for `^Wolfgang Amadeus Mozart<tab>` to find all his synonyms, and there is indeed no `Mozart`; the only two synonyms are `Johann Chrysostom Wolfgang Amadeus Mozart` and `Wolfgang Amadeus Mozart`.

I went to https://en.wikipedia.org/wiki/Wolfgang_Amadeus_Mozart, and from there to [What links here](https://en.wikipedia.org/wiki/Special:WhatLinksHere/Wolfgang_Amadeus_Mozart) and then to [http://dispenser.homenet.org/~dispenser/cgi-bin/rdcheck.py?page=Wolfgang_Amadeus_Mozart](http://dispenser.homenet.org/~dispenser/cgi-bin/rdcheck.py?page=Wolfgang_Amadeus_Mozart), and there I _do_ see `Mozart`, among many other redirects that WikipediaBase is not including as synonyms.  (And also `Johann Chrysostom Wolfgang Amadeus Mozart`, which _is_ included as a synonym.)  Is it something to do with subtypes of redirects?

@chsamlee, do you have anything to add?

**Comment by chsamlee** [comment]
I don't have anything to add.

**Comment by sfelshin** [comment]
Note that in `/archive/infolab/backup/WikipediaBase/2016-02-04/wikibase-term-synonyms`, there are dozens of synonyms for Wolfgang Amadeus Mozart.  Some of those may be bogus according to our intended algorithm, e.g., ` String Quartet No. 10`, but many are clearly good, e.g., `W.A. Mozart` and of course `Mozart`. Clearly something has gone wrong in WikiBase 1.3.0 with our code to find synonyms. For now, I'll continue to work on [START integration issues](https://github.com/infolab-csail/START/issues/39) using different synonyms, but until this problem is fixed, we can't put 1.3.0 into production use.
